deployments/hadoop-yarn/ansible/roles/prometheus_node_exporter/vars/main.yml

```
@@ -0,0 +1,6 @@
+---
+serviceName: "node_exporter"
+userId: "node_exporter"
+groupId: "node_exporter"
+exec_command: /usr/local/bin/node_exporter
+version: 0.16.0
```

Why are we installing an old version ?
https://github.com/prometheus/node_exporter/releases
https://github.com/prometheus/node_exporter/releases/latest
```suggestion
version: 1.1.2
```